expertise/config/core.py

```diff
@@ -1,53 +1,35 @@
-from collections import OrderedDict
+from collections import UserDict
 import json
 import random
-import os
+from pathlib import Path
```

Comment:
@carlosmondra It seems pathlib works with Py3.4 and above. I think we should we should put that somewhere in the documentation that we only support Py3.4 and above.

Reply:
Good point! I'll say that we support only python 3.6 and above though.